main/php5/php-fpm.initd

```diff
@@ -8,14 +8,13 @@
 
 : ${name:="PHP FastCGI Process Manager"}
 
-command="/usr/bin/php-fpm"
+command="/usr/sbin/php-fpm5"
```

Comment:
I strongly against rename the binary
so after package update I will get 2 binaries (old in old path and new)
also that will break all php5 docker images that expect that binary with old name